src/components/ha-qr-scanner.ts

```diff
@@ -90,11 +92,16 @@ class HaQrScanner extends LitElement {
                 </ha-button-menu>`
               : ""}
           </div>`
-      : html`<ha-alert alert-type="warning"
-          >${!window.isSecureContext
-            ? "You can only use your camera to scan a QR core when using HTTPS."
-            : "Your browser doesn't support QR scanning."}</ha-alert
-        >`}`;
+      : html`<ha-alert alert-type="warning">
+            ${!window.isSecureContext
+              ? this.localize("ui.components.qr_scanner.only_https_supported")
+              : this.localize("ui.components.qr_scanner.not_supported")}
+          </ha-alert>
+          ${this.localize("ui.components.qr_scanner.manual_input")}
+          <mwc-textfield
+            .label=${this.localize("ui.components.qr_scanner.enter_qr_code")}
+            @keydown=${this._qrKeyDown}
```

Comment:
Can we have a button to submit? Pressing keys requires a keyboard and that sucks on phones.

Reply:
Even better, on paste we auto submit ?